MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm

```diff
@@ -688,6 +692,10 @@ - (void)processLogBufferAfterCrash {
                                    includingPropertiesForKeys:nil
                                                       options:NSDirectoryEnumerationOptions(0)
                                                         error:&error];
+  if (!files && error) {
```

Comment:
Seems it is possible that error can be nil when file is nil. Is that why you have `if (error)` in line 820?